pkg/kube/util/names/names.go

```diff
@@ -133,7 +133,7 @@ func JobName(eJobName, podName string, podUID types.UID) (string, error) {
 	if podName == "" {
 		suffix = truncate(eJobName, 15)
 	} else {
-		suffix = fmt.Sprintf("%s-%s", truncate(eJobName, 13), truncate(podName, 13))
+		suffix = fmt.Sprintf("%s-%s", truncate(eJobName, 5), truncate(podName, 5))
```

Comment:
Why does this have to be shorter? Can you add the calculation to the comment, please?
Maybe we should only shorten, if we exceed the 63 char limit, like in `CalculateSecretName`.

I worry those pod names will be even more human-unreadable.